src/components/OrdersWidget/index.tsx

```diff
@@ -0,0 +1,180 @@
+import React, { useMemo } from 'react'
+import styled from 'styled-components'
+import { faExchangeAlt, faChartLine, faTrashAlt } from '@fortawesome/free-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+
+import Widget from 'components/layout/Widget'
+import Highlight from 'components/Highlight'
+import { tokenListApi } from 'api'
+import { getToken } from 'utils'
+import { Network } from 'types'
+import OrderRow from './OrderRow'
+
+const OrdersWrapper = styled(Widget)`
+  > a {
+    margin-bottom: -2em;
+  }
+
+  .header {
+    margin-top: 0;
+
+    h2 {
+      margin-bottom: 0.25em;
+    }
+  }
+`
+
+const ButtonWithIcon = styled.button`
+  > svg {
+    margin: 0 0.25em;
+  }
+`
+
+const CreateButtons = styled.div`
+  margin-top: 2em;
+  display: flex;
+  justify-content: center;
+  align-items: flex-start;
+
+  .strategy {
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    align-items: flex-start;
+
+    a,
+    button {
+      margin-left: 0.5em;
+    }
+  }
+`
+
+const OrdersForm = styled.div`
+  margin-top: 2em;
+  margin-left: 2em;
+
+  .ordersContainer {
+    display: grid;
+    grid-template-columns: auto;
+    margin: 2em 0 2em -3em;
+  }
+
+  .headerRow {
+    text-transform: uppercase;
+    font-weight: bold;
+    font-size: 0.75em;
+
+    > div {
+      border-bottom: 2px solid #ededed;
+      text-align: center;
+    }
+
+    & > * {
+      padding-bottom: 0.25em;
+    }
+  }
+
+  .rowContainer {
+    display: inherit;
+    grid-template-columns: 6% 8% auto 25% 12% 10%;
+    align-items: center;
+    margin: 0.25em 0;
+  }
+
+  .cell {
+    text-align: center;
+    vertical-align: middle;
+  }
+
+  .deleteContainer {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+`
+
+const OrdersWidget: React.FC = () => {
+  // TODO: only for temporary layout
+  const tokens = useMemo(() => tokenListApi.getTokens(Network.Mainnet), [])
```

Comment:
While this will of course change. In general when you have such calculations every time it's bets to do

```ts
const {DAI, USDC, TUSD, PAX} = useMemo(() => {
  const tokens = tokenListApi.getTokens(networkId)
  return {
    DAI : getToken('symbol', 'DAI', tokens)
    USDC : getToken('symbol', 'USDC', tokens)
    TUSD : getToken('symbol', 'TUSD', tokens)
    PAX : getToken('symbol', 'PAX', tokens)
 }, [networkId])
```

Reply:
Let's see if we get an agreement in these requirements, and we can continue this one: https://docs.google.com/document/d/1JGLGRtvyMPNa2w_j6tH6KVqxAIzCh91ajQzIUdR_jxU/edit

I saw the web, you are nailing it in CSS!